src/ex4/dist/RemoteItemManager.js

```diff
@@ -0,0 +1,53 @@
+class RemoteItemManager{
+    constructor() {
+        this.targetURL = 'http://localhost:8080/';
+    }
+
+    async add(text){
+       const response = await fetch(this.targetURL + `add`,{
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json'
+          },
+        method: "POST",
+        body: JSON.stringify({title:text})
+       })
+       await response.json();
+    }
+
+    async get(){
+        const response = await fetch(this.targetURL+`get`);
+        const data = await response.json();
+        return Object.values(data);
+    }
+
+    async setDone(id){
+        const response = await fetch(this.targetURL + `done/${id}`,{
```

Comment:
```suggestion
        const response = await fetch(this.targetURL + `/${id}/done`,{
```
this is the convention for actions
first have the prefix, then id and than the action that is done on this id 
same for all others